manifests/back.pp

```diff
@@ -58,6 +58,7 @@
   ~> Class['Taiga::Back::Migrate']
 
   Class['Taiga::Back::Database']
+  ~> Class['Taiga::Back::Migrate']
```

Comment:
The commit message does not align with what the commit does.

*Ordering* is already expressed on line 50-52:

https://github.com/opus-codium/puppet-taiga/blob/37cd8d42f63bebf580dd711dcb759e6473a89e2e/manifests/back.pp#L50-L52

Here, we notify `Taiga::Back::Seed` when `Taiga::Back::Database` is changed (e.g. when using a new database, it has to be seeded).  It may make sense to notify `Taiga::Back::Migrate` when `Taiga::Back::Database` is changed too, but this is a while new depedency:

```puppet
Class['Taiga::Back::Database']
~> Class['Taiga::Back::Migrate']
```